webapp/src/snippetBuilder.tsx

```diff
@@ -489,10 +516,14 @@ export class SnippetBuilder extends data.Component<SnippetBuilderProps, SnippetB
                                 {currentQuestion.errorMessage && <p className='snippet-error'>{currentQuestion.errorMessage}</p>}
                             </div>
                             {currentQuestion.hint &&
-                                <div className='snippet hint ui segment'>{pxt.Util.rlf(currentQuestion.hint)}</div>}
+                                <div className='snippet hint ui segment'>{
+                                    pxt.Util.rlf(currentQuestion.hint)
```

Comment:
how is this supposed to work? I don't see where those strings would be injected in our translation system.

Reply:
not sure. I didn't touch this in these PRs, so I'd have to check out Jacob implemented this